backend/src/sysupport/proxy.js

```diff
@@ -0,0 +1,54 @@
+import url from 'url'
+import path from 'path'
+import httpProxy from 'express-http-proxy'
+
+const subProject = '/subscription-projects'
+const subPackage = '/subscription-packages'
+const subOrder = '/subscription-orders'
+const report = '/reports'
+
+const allowedEndpoints = [subProject, subPackage, subOrder, report]
+
+function createProxy(config) {
+  return httpProxy(config.host, {
+    parseReqBody: false,
+
+    filter(req) {
+      return allowedEndpoints.includes(req.path) && req.session.timedCustomer.id
+    },
+
+    // eslint-disable-next-line max-statements
+    proxyReqPathResolver(req) {
+      const newPath = path.join(config.prefix, req.path)
+      const queryParams = url.parse(req.url, true).query
```

Comment:
Isn't this just `req.query`?

Reply:
Yes, true. Before we used `url.parse(...)` to get the query params as a string, now this is no longer needed and we can just use `req.query`.